src/tokens/functional/typography/typography.json

```diff
@@ -37,7 +37,8 @@
           "lineHeight": "{text.display.lineHeight}",
           "fontFamily": "{fontStack.sansSerif}"
         },
-        "$type": "typography"
+        "$type": "typography",
+        "$description": "Hero-style text for brand to product transition pages. Utilize Title (large) styles on narrow viewports."
```

Comment:
@lukasoppermann is there a way to add something like this to JSON but not output it as a CSS comment? I just want access to it in JSON docs 🤔

Reply:
I think that would be good. No one is looking at these CSS files really (I dont think)